src/test/java/com/google/api/codegen/testdata/ruby_version_index_library.baseline

```diff
@@ -12,4 +13,48 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+##
+# # Ruby Client for Google Example Library API ([Alpha](https://github.com/GoogleCloudPlatform/google-cloud-ruby#versioning))
+#
+# [Google Example Library API][Product Documentation]: A simple Google Example Library API.
+# - [Client Library Documentation][]
+# - [Product Documentation][]
+#
+# ## Quick Start
+# In order to use this library, you first need to go through the following steps:
+#
+# 1. [Select or create a Cloud Platform project.](https://console.cloud.google.com/project)
+# 2. [Enable the Google Example Library API.](https://console.cloud.google.com/apis/api/library)
+# 3. [Setup Authentication.](https://googlecloudplatform.github.io/google-cloud-ruby/#/docs/google-cloud/master/guides/authentication)
+#
+# ### Installation
+# ```
+# $ gem install library
+# ```
+#
+# ### Preview
+# #### LibraryServiceClient
+# ```rb
+# require "library/v1"
+#
+# library_service_client = Library::V1::LibraryServiceClient.new
+# formatted_name = Library::V1::LibraryServiceClient.book_path("testShelf-" + Time.new.to_i.to_s, project_id)
+# rating = :GOOD
+# book = { rating: rating }
+# response = library_service_client.update_book(formatted_name, book)
+# ```
+#
+# ### Next Steps
+# - Read the [Client Library Documentation][] for Google Example Library API to see other available methods on the client.
```

Comment:
This link doesn't make sense in this context. (It will be seen from within the client library docs.) Can we show this in the actual README but not here?